src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -269,6 +271,23 @@ QgsRasterLayerProperties::QgsRasterLayerProperties( QgsMapLayer *lyr, QgsMapCanv
   layout->addWidget( mMetadataWidget );
   metadataFrame->setLayout( layout );
 
+  // Temporal options
+  // Check if temporal properties of the raster layer are available and active,
+  // if not check if the layer has temporal capabilities from its provider
+  // and they are active.
+  if ( ( mRasterLayer->temporalProperties() &&
```

Comment:
I'm unsure about this logic -- there's settings here that could apply to any raster layer (a fixed time range), so I think we would always need to create the temporal widget.

Otherwise I can't see anyway for users to initially activate the temporal properties for a layer in the first place (for layers without provider side temporal capabilities, that is!)

Reply:
It's also going to lead to a crash if you keep this logic, because the call to mTemporalWidget->saveTemporalProperties may be called on a nullptr mTemporalWidget.

Reply:
Ok